packages/web/src/components/chat-listener/ChatListener.tsx

```diff
@@ -11,16 +13,17 @@ const { getAccountStatus } = accountSelectors
 export const ChatListener = () => {
   const dispatch = useDispatch()
   const accountStatus = useSelector(getAccountStatus)
+  const isGuest = useLocalStorage(GUEST_EMAIL)
```

Comment:
would we not want to use the "getIsGuestAccount" selector instead of localstorage?